### ISSUE TITLE
import with typescript

### ISSUE THREAD

**Issue by MaTiAtSIE** [kylebarron/parquet-wasm#464]
Hi, 
I try to import parquet-wasm. However, it seems that the folder structure of the package does not match the [README](https://github.com/kylebarron/parquet-wasm) file. For instance the line

```
import * as parquet from 'parquet-wasm'
```

does not work as the compiler error says "Cannot find module 'parquet-wasm' or its corresponding type declarations" (tsc 4.9.5). I installed the package with 

```
yarn add apache-arrow
yarn add parquet-wasm
```

After the installation, my typescript version was set to 4.5.5 and I manually reset the version to 4.9.5 by

```
yarn add typescript@4.9.5
```

The folder structure in {package_root}/node_modules/parquet-wasm is:
- bundler
-- parquet_wasm.d.ts
-- parquet_wasm.js
-- parquet_wasm_bg.js
-- parquet_wasm_bg.wasm
-- parquet_wasm_bg.wasm.ts
- esm
-- package.json
-- parquet_wasm.d.ts
-- parquet_wasm.js
-- parquet_wasm_bg.wasm
-- parquet_wasm_bg.wasm.ts
- node
-- package.json
-- parquet_wasm.d.ts
-- parquet_wasm.js
-- parquet_wasm_bg.wasm
-- parquet_wasm_bg.wasm.ts

So "parquet-wasm/bundler/arrow1" does not exist. What can be the problem?

**Comment by kylebarron** [comment]
> does not work as the compiler error says "Cannot find module 'parquet-wasm' or its corresponding type declarations" (tsc 4.9.5).

Can you either pin to the most recent stable version (i.e. 0.5.0) or try out this branch: https://github.com/kylebarron/parquet-wasm/pull/414? As mentioned [here](https://github.com/kylebarron/parquet-wasm/issues/308), the arrow2 implementation is no longer supported.

**Comment by MaTiAtSIE** [comment]
Thank you for the answer. Using the version 0.5.0 helped. I also had to enable 'asyncWebAssembly' in my webpack.config.js (webpack 5). 
Now I have another runtime issue which is documented [here](https://github.com/kylebarron/parquet-wasm/issues/471).

**Comment by kylebarron** [comment]
This should be fixed in 0.6.0, which was just published
